### PR TITLE
#5252 Inspect topic should be able to present java.sql.Timestamp

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5252)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-client/src/main/scala/oharastream/ohara/client/configurator/v0/package.scala [hunk 1]
```diff
@@ -282,79 +280,6 @@ package object v0 {
       .toRefiner
       .build
 
-  private[this] def toJson(value: Any): JsValue = value match {
```
Comment (jackyoh) [inline ohara-client/src/main/scala/oharastream/ohara/client/configurator/v0/package.scala:285]: 這邊刪除，是指支援所有的型態嗎？

Comment (chia7712) [reply ohara-client/src/main/scala/oharastream/ohara/client/configurator/v0/package.scala:285]: shabondi也有用到類似的邏輯，因此複製了大量package裡的程式碼到shabondi模組裡面，這邊我順便做了一下refactor，把package裡面重複的程式碼刪除

Comment (jackyoh) [reply ohara-client/src/main/scala/oharastream/ohara/client/configurator/v0/package.scala:285]: 了解。

**Comment by chia7712** [comment]
retry

**Pull request merged**